### PR TITLE
Cleanup of HTML and CSS

### PR DESCRIPTION
**Pull request by webnard**

### CODE REVIEW

File: core/sass/_colors.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+$gradGrey: linear-gradient(#b3b1aa, #757474, #b3b1aa);
+
+$gradGreen: linear-gradient(#caeba9, #7acc29, #caeba9);
+
+$gradRed: linear-gradient(#cf7c7d, #af2527, #cf7c7d);
+
+.mainText {
+	color: $byuBlue;
+    font-family: sans-serif;
+}
```
Comment (alexcdahl) [inline core/sass/_colors.scss]: I like this a lot. Why didn't we do this at the beginning?

**Comment by alexcdahl** [comment]
You've been busy. 621 additions and 691 deletions is impressive. It looks good to me. Do you want me to merge it or do you want to pully it?

**Comment by webnard** [comment]
Thanks! Let me make a small change. I didn't mean for _colors.scss to get committed.

**Comment by webnard** [comment]
And git is very generous with its line-change statistics. Most of those changes are renaming files. :sunglasses:

**Pull request closed without merging**